### PR TITLE
TripleliftBidAdapter - remove dependency on getTopWindowLocation util

### PR DESCRIPTION
**Pull request by brittanyzellman**
## Type of change

- [X] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Due to [util.getTopWindowLocation](https://github.com/prebid/Prebid.js/issues/3072) being deprecated, I updated the adapter to use bidderRequest.refererInfo instead.

## Other information
@jaiminpanchal27 @jsnellbaker 
[issue](https://github.com/prebid/Prebid.js/issues/3072)

### CODE REVIEW

File: modules/tripleliftBidAdapter.js [hunk 1]
```diff
@@ -17,12 +17,15 @@ export const tripleliftAdapterSpec = {
...
     let data = _buildPostBody(bidRequests);
 
     tlCall = utils.tryAppendQueryString(tlCall, 'lib', 'prebid');
     tlCall = utils.tryAppendQueryString(tlCall, 'v', '$prebid.version$');
-    tlCall = utils.tryAppendQueryString(tlCall, 'referrer', referrer);
+
+    if (bidderRequest && bidderRequest.refererInfo) {
+      let referrer = bidderRequest.refererInfo.referer;
+      tlCall = utils.tryAppendQueryString(tlCall, 'referrer', referrer);
+    }
```
Comment (harpere) [inline modules/tripleliftBidAdapter.js]: Looks good but should add a unit test.

**Comment by harpere** [comment]
@brittanyzellman any update here?

**Comment by brittanyzellman** [comment]
@harpere Sorry I have had to deprioritize writing a new unit test for this (I can't just add this to the existing test) since the start of Q4. I will hopefully get to it within the next week or two. Is there a date that these methods will be removed? I didnt see it in the issue.

**Comment by bretg** [comment]
@harpere - please review the new unit tests

**Pull request merged**